arm-servicefabric/2016-09-01/examples/ClusterPutOperation_exmaple_max.json

```diff
@@ -0,0 +1,264 @@
+          "queueEndpoint": "https://7623qfhwmnstw2.queue.core.windows.net/",
+          "tableEndpoint": "https://7623qfhwmnstw2.table.core.windows.net/"
+        },
+        "upgradeDescription": {
+          "overrideUserUpgradePolicy": false,
+          "forceRestart": true,
+          "upgradeReplicaSetCheckTimeout": "10675199.02:48:05.4775807",
+          "healthCheckWaitDuration": "00:05:00",
+          "healthCheckStableDuration": "00:05:00",
+          "healthCheckRetryTimeout": "00:45:00",
+          "upgradeTimeout": "12:00:00",
+          "upgradeDomainTimeout": "02:00:00",
+          "healthPolicy": {
+            "maxPercentUnhealthyNodes": 100,
+            "maxPercentUnhealthyApplications": 100
+          },
+          "deltaHealthPolicy": {
+            "maxPercentDeltaUnhealthyNodes": 0,
+            "maxPercentUpgradeDomainDeltaUnhealthyNodes": 0,
+            "maxPercentDeltaUnhealthyApplications": 0
+          }
+        }
+      },
+      "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/resRg/providers/Microsoft.ServiceFabric/clusters/myCluster",
+      "name": "myCluster",
+      "type": "Microsoft.ServiceFabric/clusters",
+      "location": "southcentralus",
+      "tags": {
+        "resourceType": "Service Fabric",
+        "clusterName": "myCluster"
+      }
+    }
+  },
+  "responses": {
+    "202": {},
+    "200": {
+      "headers": {},
+      "body": {
+        "type": "Microsoft.ServiceFabric/clusters",
+        "location": "southcentralus",
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/resRg/providers/Microsoft.ServiceFabric/clusters/myCluster",
+        "name": "myCluster",
+        "tags": {
+          "resourceType": "Service Fabric",
+          "clusterName": "myCluster"
+        },
+        "properties": {
+          "provisioningState": "Updating",
+          "clusterId": "466e7604-b8ae-4696-a91d-d7b5fcc63628",
+          "clusterCodeVersion": "5.5.149.9494",
+          "clusterState": "UpdatingUserConfiguration",
+          "managementEndpoint": "https://myCluster.southcentralus.cloudapp.azure.com:19080",
+          "clusterEndpoint": "https://warp-test-winfabrp-southcentralus.trafficmanager.net/runtime/clusters/466e7604-b8ae-4696-a91d-d7b5fcc63628",
+          "certificate": {
+            "thumbprint": "483B5DE52ECA35147B94C5E116FA7B2E34BB68F9",
+            "x509StoreName": "My"
+          },
+          "clientCertificateThumbprints": [
+            "5F3660C715EBBDA31DB1FFDCF508302348DE8E7A"
+          ],
+          "clientCertificateCommonNames": [
+            {
+              "isAdmin": true,
+              "certificateIssuerThumbprint": "5F3660C715EBBDA31DB1FFDCF508302348DE8E7A"
+            }
+          ],
+          "fabricSettings": [
+            {
+              "name": "Security",
+              "parameters": [
+                {
+                  "name": "ClusterProtectionLevel",
```

Comment:
I think name - value here should be enums, how user will know what values to send in.
These does not look like tags,